src/domain/valueObject/phpSettingValue.go

```diff
@@ -4,69 +4,65 @@ import (
 	"errors"
 	"strconv"
 	"strings"
+
+	voHelper "github.com/speedianet/os/src/domain/valueObject/helper"
 )
 
 type PhpSettingValue string
 
-func NewPhpSettingValue(value string) (PhpSettingValue, error) {
-	value = strings.Trim(value, "\"")
-	value = strings.TrimSpace(value)
-
-	settingValue := PhpSettingValue(value)
-	if !settingValue.isValid() {
-		return "", errors.New("InvalidPhpSettingValue")
+func NewPhpSettingValue(value interface{}) (settingValue PhpSettingValue, err error) {
+	stringValue, err := voHelper.InterfaceToString(value)
+	if err != nil {
+		return settingValue, errors.New("PhpSettingValueMustBeString")
 	}
+	stringValue = strings.Trim(stringValue, "\"")
+	stringValue = strings.ToLower(stringValue)
 
-	if settingValue == "on" || settingValue == "true" {
-		settingValue = "On"
+	if len(stringValue) == 0 {
+		return settingValue, errors.New("PhpSettingValueEmpty")
 	}
 
-	if settingValue == "off" || settingValue == "false" {
-		settingValue = "Off"
+	if len(stringValue) > 255 {
+		return settingValue, errors.New("PhpSettingValueTOoLong")
```

Comment:
TooLong.. that copilot would have gotten.

Reply:
Done!